packages/embark-library-manager/src/index.js

```diff
@@ -39,6 +41,63 @@ class LibraryManager {
     });
   }
 
+  installAll(callback) {
+    const installAll = async () => {
+      const useDashboard = this.npm._useDashboard;
+      this.npm._useDashboard = false;
+
+      const results = await Promise.all(
+        Object.entries(this.versions)
+          .filter(([packageName, version]) => {
+            // FIXME: will behave less than ideally if embark switches to using
+            // a dependency range for the various overridable packages instead
+            // of an exact version
```

Comment:
re: this comment: This is only an issue as long as we have ppl declare/install dependencies through `embark.json` right? Once we get to a point were people simply define the deps they need in `package.json` and after we got rid off our `NODE_PATH` hacks (which I think v5 would be a good opportunity to do so), we can most likely get rid of this here as well?

Reply:
Yes, as discussed in the meeting today, the goal is to eliminate the library manager completely, as well `"versions"` in `embark.json`. I'll change the comment to say `// NOTE: ...` instead of `FIXME`.

Reply:
Changes made.